compliance_checker/tests/test_cf.py

```diff
@@ -2945,7 +3097,7 @@ def test_polygon_geometry(self):
         dataset.variables["interior_ring"][:] = flip_ring_bits
         results = self.cf.check_geometry(dataset)
         # There should be messages regarding improper polygon order
-        assert results[0].value[0] < results[0].value[1]
+        assert results[0].value[0] > results[0].value[1]
```

Comment:
`.value[0]` should be score and `value[1]` should be the out of possible points, so the former should not exceed the latter.

Reply:
a fix is applied to the new commit